packages/superagent-wrapper/src/request.ts

```diff
@@ -146,9 +156,16 @@ const patchRequest = <
     status: StatusCode,
   ) =>
     patchedReq.decode().then((res) => {
-      if (res.status !== status) {
-        const error = res.error ?? `Unexpected status code ${String(res.status)}`;
-        throw new Error(JSON.stringify(error));
+      if (res.original.status !== status) {
+        const error = `Unexpected response ${String(
+          res.original.status,
+        )}: ${JSON.stringify(res.original.body)}`;
```

Comment:
Is `res.original.status` not already coerced to a string by the backticks?
```suggestion
        const error = `Unexpected response ${res.original.status}: ${JSON.stringify(res.original.body)}`;
```

Reply:
https://github.com/BitGo/api-ts/pull/170